### DISCUSSION TITLE
Getting Started Edge Box RPI-200 | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1729] [Q&A]
# Getting Started Edge Box RPI-200 | Seeed Studio Wiki

Getting Started Edge Controller

https://wiki.seeedstudio.com/Edge_Box_introduction/

**Comment by songpeng2008** [comment]
i am using  edgebox-rpi-200.
i am trying to install python library to run script.  this is a project using yolo v7 to detect objects. 

It looks like CPU architecture (aarch64) supports 64-bit, but  bash binary is 32-bit (elf 32-bit lsb). 
This suggests that while edgebox hardware is 64-bit capable, the operating system currently installed is 32-bit.

i failed to install pytorch becasuse the system is 32bit.
64-bit OS can handle more memory and provide better performance on some tasks especially machine learning project.
we train model using high spec GPU workstation and run trained model on edge device like industry raspberry pi or nvidia jetson.
could i reinstall oprating system 64-bit os on edgebox-rpi-200?
